lib/tasks/hello.rake

```diff
@@ -0,0 +1,3 @@
+task :hello do
+  puts 'hello'
```

Comment:
Is this needed? Please remove files not needed

Reply:
oh sorry it was for testing purposes